bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -276,6 +290,7 @@
 
 		<label>Wall Thermostat</label>
 		<description>Display of the current room temperature as well as the relative humidity in the room.</description>
+		<semantic-equipment-tag>Thermostat</semantic-equipment-tag>
```

Comment:
Same as previous comment: we might consider `ControlDevice` instead.

Reply:
Or `Display` ??

Reply:
If I interpret the semantic tag overview correctly, `Display` is rather targeted towards larger display devices such as projectors or televisions. Also the wall thermostat enables not only to display temperature/humidity, but also to set the desired temperature via a rotary knob.

You mentioned in the other comment that `Thermostat` can be used for both the "controlling" and the "executing" side of temperature control. Following this idea and also in order to align with the (virtual) room thermostats, I would therefore suggest to also use `Thermostat` here.

Reply:
Ok.